### ISSUE TITLE
can't uninstall game center on realme

### ISSUE THREAD

**Issue by Ansuman25** [0x192/universal-android-debloater#882]
In Realme x2 pro

I tried too many attempts but the Game Center pre-installed app is not disable/uninstall

**Comment by SKS59** [comment]
Yes same issue for me. I am not able to uninstall theme store along with game center

**Comment by aisivan36** [comment]
Some of the system apps cannot be uninstalled through adb method, the only way is to disable the package that cannot be uninstalled.

**Comment by moody-jazz** [comment]
yes that is the only issue other than that everything works fine
